### DISCUSSION TITLE
writing Usage article

### DISCUSSION THREAD

**Discussion by jhk0530** [morethanmin/morethan-log#76] [General]
This service looks **AWESOME** to me, So I hope to write and share an article about utilizing this. 🤩

Is there any guide or things that I should notice before? 

*(or maybe I can share draft before publish)*

Thanks.

@morethanmin

**Comment by morethanmin** [comment]
Thanks for your interest in morethan-log ☺️

It's enough just to share published link for me!

**Reply by jhk0530** [reply]
Thanks

**Comment by jhk0530** [comment]
I wrote an [article](https://jhk0530.medium.com/notion-blog-with-morethanlog-3f4409e6f613) ! (**in Korean**). 

I'll share If I write more articles or modify them.

Thanks again for the nice product 🤗

**Comment by KRRKireeti** [comment]
Hello @jhk0530 ,
Can you please clarify one of my doubts?

DO we need to redeploy the project in Vercel each time we make changes to the Notion page for them to reflect in the blog side?

**Reply by jhk0530** [reply]
As far as I know, Nope.
